### DISCUSSION TITLE
using EAM output from a previous run to run a ELM case

### DISCUSSION THREAD

**Discussion by pgoddard08** [E3SM-Project/E3SM#6401] [E3SM model help]
Hello, I'm looking for guidance on how to use EAM output from a previous run to serve as initial and boundary conditions to run a ELM case (I think this is referred to as a land model run with a data driven atmosphere). Can someone point me to a good place to begin as I work to better frame my question?

**Comment by pgoddard08** [comment]
Hello, I posed a similar question to the CESM discussion board in hopes that I could gather some insights from how this is done in CESM. Here is that blog:
https://bb.cgd.ucar.edu/cesm/threads/using-cam-output-from-a-previous-run-to-run-a-clm-case.9612/

Is there anyone familiar with doing such an experiment in E3SM and would be willing to be a point of contact for me? Thank you.

**Reply by jonbob** [reply]
My name is Jon Wolfe and I'm a LANL E3SM person. I have done several runs using the CPLHIST files to force other runs and I can help you

**Reply by pgoddard08** [reply]
Hi Jon,
I am Paul Goddard, an Assistant Research Scientist at Indiana University. 

As part of a large ensemble hindcast project, I need use the atmo output from an AMIP run (compset F20TR) to force a land model run (I'm not sure which compset to use; I can check with my PI or hear what you suggest). 

As mentioned in the blog post above, CESM help desk pointed me towards https://escomp.github.io/ctsm-docs/versions/master/html/users_guide/running-special-cases/Running-with-your-own-previous-simulation-as-atmospheric-forcing-to-spinup-the-model.html

Is this similar to the protocol you follow? If so, I am confused on how to proceed, or what to edit for "edit the driver code in order to save the correct list of fields (see note below)".  

I appreciate your guidance here. I am new to running E3SM and ESMs.

**Reply by jonbob** [reply]
Hi Paul,

Do you know what kind of ELM run you want to make? I'm assuming it's some sort of an I-case and not a standalone run. If so, that would mean your forced run would have active ELM and MOSART, so we mostly have to be concerned about running datm (the data atm model) in its CPLHIST mode, where the CPL HIST files will be generated by your AMIP run.

Assuming that's true, it's relatively easy to tell the AMIP run to output the files needed for the ELM run. All you really need to do is add these lines to user_nl_cpl in your AMIP case before you run it:
````
histaux_a2x1hri = .true.
histaux_a2x3hr = .true.
histaux_a2x1hr = .true.
histaux_a2x24hr = .true.
````
Those settings just tell the E3SM coupler to output auxiliary history files that can get used by datm as cplhist files. As a warning, it will generate a lot of small files, so you may want to run just a year or two at a time. But that depends on how long the AMIP run is.

There is an intermediate step to concatenate these files, which are output daily, into the format datm requires. I can help you with that once you have generated the files.

For the I-case run, there are some settings to tell datm to run in CPLHIST mode, as well as to point to the output from the AMIP run. But first things first...

**Reply by pgoddard08** [reply]
Hi Jon,

Great, thanks for direction. I'll work on the AMIP run. 

We will be running an I-case, I think it will be I20TRELM with geochemistry crops (perhaps that's I20TRELMCBCN?).  

I also wanted to ask what it takes to run an I-case with the datm forced from reanalysis data, say NCEP CFS. Is this something you are familiar with too?

**Reply by jonbob** [reply]
Hi Paul,

I'm not a land modeler -- but the datm model is configured to use a bunch of different forcings. I'm more familiar with the ones the ocn/ice teams tend to use, like CORE-II and JRA. ELM I-compsets tend to use NCEP CRU, QIAN, or GSWP forcing, but it looks like datm also has an option for CFS.

Jon

**Comment by pgoddard08** [comment]
Thanks Jon.  To clarify, if I want to run an I-case with NCEP CRU forcing I would use one of the available I-compsets (e.g., ICRUELM) - with no other modifications?  (If so, I did not see a I-compset using CFS for E3SM v2.1.1 -- but perhaps I just missed it in the list).

**Reply by jonbob** [reply]
Exactly -- the NCEP CRU forcing has a pre-defined compset you can use. And there is not one for CFS -- as far as I can tell, that forcing is currently only used by the wave model. It's likely then that the CFS forcing data the data atm model knows about is fairly limited, and might take a good bit of work to be useable by ELM.

**Reply by pgoddard08** [reply]
Great - thanks for the information.  My PI and I are reviewing our methodology and your help is greatly appreciated.
